From 7ec60384809a9e056d6cf260f6fd3585623981b4 Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Sat, 17 Sep 2022 12:15:55 +0000
Subject: [PATCH] changes for ArrowOS Makefile

---
 base-pre.mk |  2 +-
 base.mk     | 29 +++++++++++++++++++++++++----
 2 files changed, 26 insertions(+), 5 deletions(-)

diff --git a/base-pre.mk b/base-pre.mk
index 6a317e4..ae5b289 100644
--- a/base-pre.mk
+++ b/base-pre.mk
@@ -1,3 +1,3 @@
 #Use a more decent APN config
 PRODUCT_COPY_FILES += \
-	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
+#	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
diff --git a/base.mk b/base.mk
index af27de5..221bfce 100644
--- a/base.mk
+++ b/base.mk
@@ -8,10 +8,6 @@ PRODUCT_COPY_FILES := \
 	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
 
-#Use a more decent APN config
-PRODUCT_COPY_FILES += \
-	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
-
 BOARD_PLAT_PRIVATE_SEPOLICY_DIR += device/phh/treble/sepolicy
 PRODUCT_PACKAGE_OVERLAYS += device/phh/treble/overlay
 
@@ -50,6 +46,7 @@ PRODUCT_COPY_FILES += \
 #   Provide default libnfc-nci.conf file for devices that does not have one in
 #   vendor/etc
 PRODUCT_COPY_FILES += \
+	system/nfc/conf/libnfc-nci.conf:system/etc/libnfc-nci.conf \
 	device/phh/treble/nfc/libnfc-nci.conf:system/phh/libnfc-nci-oreo.conf \
 	device/phh/treble/nfc/libnfc-nci-huawei.conf:system/phh/libnfc-nci-huawei.conf
 
@@ -233,8 +230,32 @@ PRODUCT_COPY_FILES += \
 	frameworks/av/services/audiopolicy/config/a2dp_audio_policy_configuration_7_0.xml:system/etc/a2dp_audio_policy_configuration_7_0.xml \
 	frameworks/av/services/audiopolicy/config/a2dp_audio_policy_configuration.xml:system/etc/a2dp_audio_policy_configuration.xml \
 
+PRODUCT_PACKAGES += \
+	android.hardware.vibrator@1.3-service.meizu
+
 include build/make/target/product/gsi_release.mk
 
 # Protect deskclock from power save
 PRODUCT_COPY_FILES += \
 	device/phh/treble/files/com.android.deskclock_whitelist.xml:system/etc/sysconfig/com.android.deskclock_whitelist.xml
+
+$(call inherit-product, vendor/arrow/config/common.mk)
+$(call inherit-product, vendor/arrow/config/BoardConfigSoong.mk)
+$(call inherit-product, vendor/arrow/config/BoardConfigArrow.mk)
+$(call inherit-product, device/arrow/sepolicy/common/sepolicy.mk)
+$(call inherit-product, vendor/arrow/config/bootanimation.mk)
+-include vendor/arrow/build/core/config.mk
+TARGET_NO_KERNEL_OVERRIDE := true
+TARGET_NO_KERNEL_IMAGE := true
+SELINUX_IGNORE_NEVERALLOWS := true
+TARGET_BOOT_ANIMATION_RES := 1080
+
+TARGET_USES_PREBUILT_VENDOR_SEPOLICY := true
+TARGET_HAS_FUSEBLK_SEPOLICY_ON_VENDOR := true
+
+TARGET_FACE_UNLOCK_SUPPORTED := true
+
+PRODUCT_PACKAGES += \
+    GCamGOPrebuilt
+
+#ARROW_GAPPS := true
-- 
2.25.1

